content/ui-patterns/saving.mdx

```diff
@@ -0,0 +1,222 @@
+</Box>
+
+## Calls to action
+
+When data is not automatically saved after the user make changes, buttons are used to submit or cancel the changes.
+
+### Text
+
+All configuration flows should have calls to action that include an obvious active verb such as "Create", "Save", "Delete", or "Update" (for example, "Create" a new repository, "Add" an SSH key to your profile, "Save" security preferences, "Update" a repository's description, "Delete" an old email address).
+
+When defining these calls to actions, make sure to:
+1. Keep the text as succinct and clear as possible
+2. Add the item’s name to the button text in cases where it may be unclear to the user what is being changed
+
+### Appearance
+
+If the save button is used to save every input on the page, use the _primary_ button appearance.
+
+If the save button has a corresponding cancel button, use the _primary_ button appearance for the save button, and _secondary_ for the cancel button.
+
+If the save button only refers to a segment of inputs on the page, use the _secondary_ button appearance.
+
+For more guidance on button usage, see the [button interface guidelines](/ui-patterns/button-usage).
+
+### State
+
+Do not disable or hide a form's save button even if the form is invalid or has not been changed. Disabled buttons cannot be focused using the Tab key, and disabled button styles are typically low-contrast and difficult to read.
+
+### Placement
+
+Place save buttons somewhere intuitive and easily accessible. If all of the fields save automatically, do not include a save button. This makes it unclear when data is being saved. See the guidelines on [explicit saving](#explicit-saving) and [automatic saving](#automatic-saving) for more information.
+
+There should only be one save button on a page. Allow only one edit mode activated at a time if you're designing a page with content that can be edited separately (such as an issue title). Pages that contain multiple save buttons can cause confusion about which save button saves which group. Multiple save buttons with the same label can cause confusion for people who use assistive technologies like screen readers and voice recognition software.
+
+<DoDontContainer>
+  <Do>
+    <img alt="A form with one save button at the bottom" src="https://user-images.githubusercontent.com/2313998/151887934-cfcc5130-53a8-4f40-ba36-97148ead72c0.png" />
+    <Caption>Always keep the save button enabled</Caption>
+  </Do>
+  <Dont>
+    <img alt="A form with two sections. Each section has it's own save button." src="https://user-images.githubusercontent.com/2313998/151887938-1fa6b5d7-c5c1-4010-ab99-e5c37bff8c0a.png" />
+    <Caption>Disable the save button if the form is invalid or unchanged</Caption>
+  </Dont>
+</DoDontContainer>
+
+<img width="960" src="https://user-images.githubusercontent.com/2313998/151448415-24fc9f15-627f-47b7-abab-2020d285b770.png" />
```

Comment:
This is an image with toggle buttons that don't exist yet; should we remove it?

Reply:
Oops! Yes, that should have been removed